src/views/routes/users.js

```diff
@@ -61,16 +65,61 @@ router.post('/login', async (req, res) => {
   }
 });
 
+router.get('/:username', async (req, res) => {
```

Comment:
TODO - CREATE TICKET: This was an old endpoint which is used by the front-end as an async validation to check if the user has already registered with the username passed as the request parameter. The logic in the frontend should be updated to accommodate these changes.